index.html

```diff
@@ -14,6 +14,15 @@ <h1>Portsmouth Degree&nbsp;Calculator</h1>
 
 <main>
 
+  <section id="up-number-input">
+    <h2>Input your UP number to get your data:</h2>
+    <div class="input-container">
+      <input type="text" id="up-input" required="UP">
+      <label for="input" class="label">Enter UP number:</label>
```

Comment:
Also, I think there's an overkill of information here. Like you have a header with "Input your UP number to get your data:" and then you say again the same thing in the input "Enter UPnumber". I would say this instead:
```<label for="input" class="label">UPXXXXXX</label>```